install_dep.sh

```diff
@@ -0,0 +1,23 @@
+# check that all the dependecies are installed
```

Comment:
This should start with `#!/bin/bash` or `#!/bin/sh`. Also consider adding `set -ex` to print the commands and exit when any of them fails.

Reply:
Where should I add `set -ex`?

Reply:
It usually goes on the second line, after `#!/bin/bash`. Keep in mind that I'm not saying this is the right choice, I'm just providing an option.

Reply:
Seems good to me